src/badge/styled-components.ts

```diff
@@ -184,8 +184,6 @@ export const StyledRoot = styled<'div', {}>('div', () => {
   };
 });
 
-StyledRoot.displayName = 'StyledRoot';
```

Comment:
Please keep the displayName assignments. Enzyme uses this property to search for components in the react tree

Reply:
Ah. My understanding was that the display name is automatically deduced from the name of the function (or the variable the component is assigned to), and thus explicitly assigning displayName is only necessary when you want the component to show up differently in the dom. Is that incorrect?

Reply:
Not when building with styletron :(

Reply:
Gotcha. Added displayName to all the recent new components.